docker-compose-e2e.yaml

```diff
@@ -0,0 +1,426 @@
+      retries: 5
+      start_period: 30s
+    depends_on:
+      - zgw-referentielijsten-database
+
+  # Insert required test data into the VRL database after the VRL application has started up and finished initializing the database
+  zgw-referentielijsten-database-init:
+    image: postgres:15.4
+    container_name: zgw-referentielijsten-database-init
+    volumes:
+      - ./scripts/docker-compose/imports/zgw-referentielijsten/init-zgw-referentielijsten-database.sql:/tmp/init-zgw-referentielijsten-database.sql
+    command: bash -c "psql postgresql://vrl:vrl@zgw-referentielijsten-database/vrl < /tmp/init-zgw-referentielijsten-database.sql"
+    depends_on:
+      zgw-referentielijsten:
+        condition: service_healthy
+
+  # The GBA mock is required by the BRP proxy
+  gbamock:
+    image: ghcr.io/brp-api/haal-centraal-brp-bevragen-gba-mock:2.0.8
+    platform: linux/amd64
+    container_name: gbamock-e2e
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Release
+      - ASPNETCORE_URLS=http://+:5010
+    ports:
+      - "5010:5010"
+
+  brpproxy:
+    image: ghcr.io/brp-api/haal-centraal-brp-bevragen-proxy:2.1.0
+    platform: linux/amd64
+    container_name: brpproxy-e2e
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Release
+      - ASPNETCORE_URLS=http://+:5000
+    ports:
+      - "5001:5000"
+    depends_on:
+      - gbamock
+
+  openklant-database:
+    image: postgres:15.4
+    container_name: openklant-database-e2e
+    ports:
+      - "54325:5432"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER}"]
+      interval: 20s
+      timeout: 60s
+      retries: 5
+      start_period: 30s
+    environment:
+      - POSTGRES_USER=openklant
+      - POSTGRES_PASSWORD=openklant
+      - POSTGRES_DB=openklant
+    volumes:
+      - ./scripts/docker-compose/imports/openklant-database:/docker-entrypoint-initdb.d
+      - ./scripts/docker-compose/volume-data/openklant-database-data:/var/lib/postgresql/data
+
+  openklant:
+    image: docker.io/maykinmedia/open-klant:latest
+    container_name: openklant-e2e
+    platform: linux/amd64
+    environment:
+      - ALLOWED_HOSTS=localhost,host.docker.internal,172.17.0.1,openklant
+      - DB_HOST=openklant-database
+      - DB_USER=openklant
+      - DB_PASSWORD=openklant
+      - DB_NAME=openklant
+      - CACHE_DEFAULT=redis:6379/0
+      - CACHE_AXES=redis:6379/0
+      - DJANGO_SETTINGS_MODULE=openklant.conf.docker
+      - IS_HTTPS=no
+      - NOTIFICATIONS_DISABLED=True
+      - SECRET_KEY=openZaakSecretKey
+    ports:
+      - "8002:8000"
+    depends_on:
+      openklant-database:
+        condition: service_healthy
+      redis:
+        condition: service_started
+
+  zac:
+    # you typically want to change this to the most recent version of the ZAC Docker Image
+    # see: https://github.com/infonl/dimpact-zaakafhandelcomponent/pkgs/container/zaakafhandelcomponent
+    image: ghcr.io/infonl/zaakafhandelcomponent:main-242
```

Comment:
This is an old version of ZAC. With the latest changes this should now probably be:
```
image: ghcr.io/infonl/zaakafhandelcomponent:main-242
```

Reply:
Do you mean `:latest` @edgarvonk ?

Reply:
Indeed! :-)

Reply:
In the end you should use the Docker image that was built as part of the current build I think and not the latest (that's already old). But for now let's first get it working with 'latest'.

Reply:
PS: I will be doing this (using the Docker image that was built in the current build flow) in my upcoming PR for the integration tests so we can later on probably reuse that here.